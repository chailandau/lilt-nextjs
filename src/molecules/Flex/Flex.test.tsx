import { composeStory } from '@storybook/react';

import Meta, { Default } from './storybook/Flex.stories';

import {
    testAxeViolations,
    testMatchesSnapshot,
    testPropOptions
} from '@/utils/testHelpers';

const Flex = composeStory(Default, Meta);
const flexComponent = {
    component: <Flex />
};

describe('Flex', () => {
    testPropOptions({
        ...flexComponent,
        propName: 'className',
        propOptions: ['test'],
        htmlTag: 'div'
    });
    testAxeViolations(flexComponent);
    testMatchesSnapshot(flexComponent);
});
