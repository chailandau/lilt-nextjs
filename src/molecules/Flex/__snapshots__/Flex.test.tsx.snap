// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex"
  >
    <p
      class="text md"
    >
      This text is in a flex container
    </p>
  </div>
</DocumentFragment>
`;
